.babelrc

```diff
@@ -1,35 +1,24 @@
 {
-  presets: [
-    [
-      "env",
-      {
-        targets: {
-          node: "current",
-          browsers: "last 2 chrome versions, last 2 firefox versions"
-        },
-        modules: false
-      }
-    ],
-    "react",
-    "flow"
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react",
+    "@babel/preset-flow"
   ],
-  plugins: [
-    "transform-class-properties",
-    [ "transform-object-rest-spread", { "useBuiltIns": true }]
+  "plugins": [
+    ["@babel/plugin-proposal-class-properties", { loose: true }],
+    [ "@babel/plugin-proposal-object-rest-spread", { "useBuiltIns": true } ]
   ],
-  env: {
-    // Needed for tests, as node doesn't support ES2015 modules yet.
-    // The version of node we use doesn't need the generator polyfill so we're not including it
-    test: {
-      presets: [
-        [ "env", { target: { node: "current" }} ],
-        "react",
-        "flow"
-      ],
-      plugins: [
-        "transform-class-properties",
-        [ "transform-object-rest-spread", { "useBuiltIns": true }]
+  "env": {
+    "test": {
+      "presets": [
+        "@babel/preset-env",
+        "@babel/preset-react",
+        "@babel/preset-flow"
       ],
+      "plugins": [
+        ["@babel/plugin-proposal-class-properties", { loose: true }],
+        [ "@babel/plugin-proposal-object-rest-spread", { "useBuiltIns": true } ]
+      ]
```

Comment:
the default and the "test" envs are now identical, but I'm keeping them both because I know Markus will introduce new divergences in his "demangling with wasm" patch.